samples/replicated/shopping-cart-service-scala/kubernetes/deployment.yml

```diff
@@ -58,7 +57,36 @@ spec:
             - name: REQUIRED_CONTACT_POINT_NR
               value: "1"
             - name: JAVA_TOOL_OPTIONS
-              value: "-XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75"
+              value: "-XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75 -Dakka.persistence.r2dbc.connection-factory.ssl.enabled=true -Dakka.persistence.r2dbc.connection-factory.ssl.mode=require"
+            - name: SELF_REPLICA_ID
+              valueFrom:
+                secretKeyRef:
+                  name: replication-secret
```

Comment:
easiest to extract those to a secret so that the same deployment.yml can be used in both places